taskchampion/src/task/task.rs

```diff
@@ -443,7 +443,9 @@ impl<'r> TaskMut<'r> {
         value: Option<String>,
     ) -> anyhow::Result<()> {
         let property = property.into();
-        self.lastmod()?;
+        if &property != "modified" {
+            self.lastmod()?;
```

Comment:
Maybe worth renaming `lastmod` to `update_lastmod` or `set_lastmod`? Reads a little confusingly as sounds like it's an getter

Reply:
Also maybe worth a few more comments (e.g I assume the `if property != "modified"` check is to avoid recursion with lastmod being updated when setting last modified, but would probably be worth explicitly writing down)

Reply:
Fixed in 393b804b30f062bd3fccae0199bfad55e4540161